js/src/Geonotebook.js

```diff
@@ -90,22 +108,6 @@ Geonotebook.prototype.recv_msg = function (message) {
     // Once protocol negotiation is complete create the geojs map
     // and add the base OSM layer
     this.map.init_map();
-
-    this.map.notebook._remote.get_map_state().then((state) => {
-      _.each(_.union(state.layers.system_layers, state.layers.layers), (layer) => {
-        this.map.add_layer(layer.type, layer.name, layer.kwargs);
-
-        if (layer.type === 'annotation' && _.size(layer.annotations)) {
-          _.each(layer.annotations, (annotation) => {
-            this.map.add_annotation(annotation.type, annotation.args, annotation.kwargs);
-          });
-        }
-      });
-
-      if (_.has(state, 'center')) {
-        this.map.set_center.apply(this.map, state.center);
-      }
-    });
```

Comment:
You remove this chunk into its own function (which is fine), but never call it from here. So refreshing breaks everything.